repo/packages/M/minio/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "minio",
+  "version": "0.0.1-RELEASE.2016-09-11T17-42-18Z",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/minio/minio.git",
+  "maintainer": "support@mesosphere.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server compatible with Amazon S3. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images.",
+  "tags": ["data", "storage", "filesystem"],
+  "preInstallNotes": "Installing Minio.",
```

Comment:
we will remove this once its being used by the community and when we add some automated testing to the package.